src/utils.test.ts

```diff
@@ -67,6 +79,10 @@ describe("parseSkylink", () => {
     });
   });
 
+  it("should parse out base32 skylink from subdomain", () => {
+    expect(parseSkylink(portalUrlSubdomain, { subdomain: true })).toEqual(skylinkBase32);
```

Comment:
I would expect `parseSkylink` to work like this:

```suggestion
    expect(parseSkylink(portalUrlSubdomain)).toEqual(skylink);
```